.github/workflows/e2e/dapps/v2explorer.yaml

```diff
@@ -0,0 +1,31 @@
+name: v2explorer-android
+
+on:
+  push:
+    branches: [ feat/E2E-maestro ]
+  # pull_request:
+  #   branches: [ main ]
+
+jobs:    
+  maestro-cloud:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: dapps/v2Explorer
+    outputs:
+      app: app/build/outputs/apk/debug
```

Comment:
```suggestion
      app: app/build/outputs/apk/release
```

Reply:
That was intentional, I thought it would be better to test the debug version for a more verbose output
I can switch it to release if that makes more sense?